package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@google-cloud/bigquery-storage",
+  "version": "0.1.0",
+  "description": "Client for the BigQuery Storage API",
+  "repository": "googleapis/nodejs-bigquery-storage",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "files": [
+    "build/src",
+    "build/protos"
+  ],
+  "main": "build/src/index.js",
+  "scripts": {
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "docs": "jsdoc -c .jsdoc.js",
+    "predocs-test": "npm run docs",
+    "docs-test": "linkinator docs",
+    "fix": "gts fix && eslint samples --fix",
+    "lint": "gts check && eslint samples",
```

Comment:
If you add the line:

```
    "prelint": "cd samples; npm link ../; npm i"
```

I think you will find things start working for you, as soon as you fix the version #.